spec/models/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it 'should have valid Factory' do
+    expect(create(:user)).to be_valid
+  end
```

Comment:
IF you decide to follow my advice and add a second factory then please add another test here.

Reply:
ok

Reply:
add test for creating `: journalist` with factorybot here